auth_brute_force/views/view.xml

```diff
@@ -10,6 +10,7 @@
             <tree
                 decoration-warning="result == 'failed'"
                 decoration-danger="result == 'banned'"
+                delete="0" create="0"
```

Comment:
Permissions already do this, unless you're an admin, in which case IMHO you should be able to at least delete things.

Reply:
I think it is a bad idea, because as an admin, you will be deleting accesses to your system. I am not sure if it is right by GDPR.

Reply:
Nothing in the GDPR explicitly forces you to record all logins AFAIK. And you can achieve this better if you use case requires that through permissions.

Reply:
Well, maybe there is no detail on GDPR, but if you had an attack from a whitelisted remote, you could delete the attack history in order to clean your responsibilities. Do you think that option is GDPR compliant?

Reply:
As an admin, you can uninstall the addon, which deletes all of that, and reinstall it, so that's not a valid argument AFAICS.

Reply:
True, I will remove the options.